src/parallel_specs.inp

```diff
@@ -1,25 +1,34 @@
-function void display_workers_output (const int n_jobs,
-                                      const bool debugging,
+function void display_workers_output (int n_jobs,
+                                      bool debugging,
                                       scalar *error)
     /* Print results of workers. */
 
-    loop i=1..n_jobs -q
-        printf "Info: Output for argument set %d:\n", i
+    loop i = 1..n_jobs -q
+        printf "PS-Info: Output for argument set %d:\n", i
```

Comment:
Hi Sven, why did you change "Info" to "PS-Info"?
I really recommend to follow standards here and that is usually the "Info" keyword for logging purpose. The reason is simple: filter large log-files by some standard keyword such as "Info", "Error" etc.

Reply:
Hi Sven, thanks for the changes. As you see I have only a single comment.

One word on running tests: You can always run the ./tests/run_tests.inp file as every gretl script. You just need to set the working directory on your machine.

The tests run ok here.